js_sdk/install_glia_with_site_id.js

```diff
@@ -0,0 +1,18 @@
+var installGlia = function(siteId, callback) {
+  var gliaIntegrationScriptUrl = 'https://api.salemove.com/salemove_integration.js?site_id=' + siteId;
```

Comment:
const gliaIntegrationScriptUrl = `https://api.salemove.com/salemove_integration.js?site_id=${siteId}`;

Reply:
@paniagua I asked him to change it as it's now because `const` and template literals are not supported in all browsers (IE11), so it's a bad idea to use it directly.

Reply:
also `var` is likely safer for direct copy paste scripts even though ie11 supports it